app/views/pages/home.html.erb

```diff
@@ -152,22 +152,22 @@
           <div class="col-sm-4">
             <div class="inner-working">
               <h2>Step 1</h2>
-              <h3 id="how-it-works">Facebook Feed Data Parsing</h3>
-              <p id="how-it-works">We're looking at your Facebook feed thanks to our Chrome Extension and looking at all the news links we see on there.</p>
+              <h3 id="how-it-works">Facebook Newsfeed Data Parsing</h3>
+              <p id="how-it-works">Our Chrome Extension keeps track of all the news links (and only the news links!) from your Facebook newsfeed and the amount of friends who like certain political Facebook pages.</p>
```

Comment:
Maybe we can use this wording "...the number of friends (without capturing any names) who like certain political Facebook pages"